src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c

```diff
@@ -1036,6 +1090,11 @@ static void ert_intc_enable(struct xocl_ert_user *ert_user, bool enable){
 	uint32_t i;
 	xdev_handle_t xdev = xocl_get_xdev(ert_user->pdev);
 
+	if (XOCL_DSA_IS_VERSAL(xdev)) {
+		xocl_mailbox_versal_request_intr(xdev, ert_versal_isr, ert_user);
+		return;
+	}
+
```

Comment:
you don't need disable in versal?

Reply:
The mailbox_versal subdev will disable and enable the interrupt. This api only register the isr onto it.